docs/en/status-communities/how-to-use-status-communities-your-quick-start-guide.md

```diff
@@ -0,0 +1,102 @@
+
+In Status, community owners have complete control over how users join their communities and can restrict access [using tokens][token-based-access-to-communities-and-channels].
+
+In token-gated communities, members must own and maintain the required tokens to join and participate in the community. In addition to token requirements, community owners may require [manual approval to join][about-community-request-approvals] their communities.
+
+![A screen of the Status mobile app showing a community with token requirements. To access this community, you must fulfil the token requirements.](./how-to-use-status-communities-your-quick-start-guide/110-0-3.png)
+
+=== ":mobile-one: Token requirements"
+
+    Token-gated communities require specific tokens and token amounts to join. When you hold the necessary tokens, a :mobile-green-check: green checkmark appears next to the token. Check out [Understand token requirements in communities][understand-token-requirements-in-communities] for more information.
+
+=== ":mobile-two: Join community"
+
+    In token-gated communities, the :mobile-unlocked-lock: **Join community** option is active only when you meet the token requirements.
+
+!!! note
+
+    When you join a token-gated community, your tokens work as proof of ownership and stay in your Status Wallet. They are not transferred to the community owner, Status or someone else.
+
+Status Communities contain [channels][channels-your-quick-start-guide]. Think of channels as rooms, each with a unique theme, helping keep discussions focused and organized.
+
+Community owners can also control [access to channels using tokens][create-a-token-gated-channel] or other permissions. Some channels might be open to all members, while others can be restricted or read-only.
+
+![A screenshot of a token-gated channel with different token-based permission for the view-only and view and post access](./how-to-use-status-communities-your-quick-start-guide/110-0-4.png)
+
+=== ":mobile-one: View only requirements"
+
+    Token-gated channels require specific tokens and token amounts to join and read the channel. When you hold the necessary tokens, a :mobile-green-check: green checkmark appears next to the token.
+
+=== ":mobile-two: View and post requirements"
+
+    Owners can configure different view-only and view and post requirements for channels. 
+
+=== ":mobile-two: Enter channel"
+
+    If you meet some or all of the channel requirements, the :mobile-unlocked-lock: **Enter channel** option is active and you can join the conversation.
+
+!!! tip
+    Use your [notification settings][configure-community-and-channel-notifications] to customize the community and channel notifications.
+
+## Communities' privacy and security
+
+When you participate in a channel, your messages and profile are visible to anyone with access to the same channel. You can customize [what information others can see][view-your-profile-as-someone-else] in your profile.
```

Comment:
I believe profile is visible to anyone, not just who has access to that channel

Reply:
You're right, @cammellos – I've fixed that sentence. Thanks!